prospector/README.md

```diff
@@ -93,6 +93,14 @@ Try the following example:
 
 To run the tests, run:
 
+## Testing
+
+First, you need to install pytest-cov:
+
+`pip install pytest-cov`
```

Comment:
This is not needed, it's installed with `pipenv install --dev`

Reply:
Sorry this keeps appearing after rebase.